### ISSUE TITLE
handling 2 liaisons

### ISSUE THREAD

**Issue by kenirwin** [Miamiohlibs/Dashboard#4]
What happens if there are two liaisons for a subject? (Marketing: Abigail Morgan, Senta Sellers)

**Comment by kenirwin** [comment]
Demo user: "fakeUsers/marketing.json"

```
{
    "user": "markstu",
    "attributes": {
        "displayName": [
            "Markeing Stu"
        ],
        "muohioeduCurrentCourseSubjectNumber": [
            "MKT101"
        ],
        "muohioeduPrimaryAffiliation": [
            "student"
        ],
        "uid": [
            "markstu"
        ],
        "eduPersonPrimaryAffiliation": [
            "student"
        ],
        "muohioeduPrimaryLocationCode": [
            "oxf"
        ],
        "eduPersonPrincipalName": [
            "markstu@miamioh.edu"
        ],
        "muohioeduMajorCode": [],
        "givenName": [
            "Mark"
        ],
        "muohioeduMajor": [],
        "muohioeduPrimaryLocation": [
            "Oxford"
        ],
        "muohioeduPrimaryAffiliationCode": [
            "und"
        ]
    }
}
```

**Comment by kenirwin** [comment]
@Meng-MiamiOH -- I'm wondering if you can take a look at this. I'm working on the problem of displaying two librarians where relevant, and getting Bootstrap to toggle open/closed the "extra" information from LibGuide Widgets. 

The code that I'm generating works outside the Dashboard, like in this codePen test: 
https://codepen.io/kenirwin/pen/XWMjpyJ
(click the "more" buttons to hide or show more content)

But the same code in the Dashboard isn't working. To set up: 
* check out the branch "two-librararians"
* in the `fakeUsers` folder, add a "marketing.json" file with the content from the previous comment
* in `config/fakeUsers.json` add and select the fake user, e.g.:
```
{
    "useFakeUser": true,
    "fakeUserId": "6",
    "fakeUsers": [
        "ken.json",
        "mjf-fac.json",
        "3822.json",
        "employee-in-class.json",
        "fac.json",
        "engine-undergrad.json",
        "marketing.json"
    ]
}
```
* then `./restart`

**Comment by Meng-MiamiOH** [comment]
@kenirwin In your codepen demo, I saw that you were using Bootstrap 4.6 which is an older version. I have upgraded Bootstrap to version 5 in order to use the component "offcanvas".

You may modify the collapse feature according to the latest guide at https://getbootstrap.com/docs/5.0/components/collapse/

**Comment by kenirwin** [comment]
Thanks, Meng! That worked.


<http://miamioh.edu/>

Ken Irwin

Web Services Librarian

Miami University

King Library 302

151 S. Campus Avenue

Oxford, OH 45056

O: 513-529-4212 | irwinkr@MiamiOH.edu *Pronouns: he/him/his *

**Comment by kenirwin** [comment]
Merged in 63dd044
